cmd/wallet-sdk-gomobile/api/api.go

```diff
@@ -58,13 +58,14 @@ type KeyHandleReader interface {
 type CreateDIDOpts struct {
```

Comment:
please follow this pattern : https://github.com/trustbloc/agent-sdk/blob/main/cmd/wallet-js-sdk/src/did/didmanager.js#L71-L181

Reply:
I need some clarification on which of those parameters applies here, since that `didmanager.js` method is specific to `did:orb`, while this `CreateDIDOpts` is more generic. Also, please see here: https://github.com/trustbloc/wallet-sdk/pull/30#discussion_r1008310488 I also need some clarification on how this DIDCreator interface should work.

Reply:
Updated based on our discussion earlier.